tests/ui/typeck/issue-114423.rs

```diff
@@ -0,0 +1,15 @@
+struct RGB {
```

Comment:
Just realized the file name. We're moving away from having no context beyond the issue number in test names. Would you mind changing it to something like `issue-114423-ice-regression-in-suggestion.rs`?